AdyenSession/AdyenSession.swift

```diff
@@ -155,7 +163,7 @@ extension AdyenSession {
     
     // TODO: remove adyen context and action config from this
     /// Session configuration.
-    public struct Configuration {
+    public struct SetupModel {
```

Comment:
Maybe we can rename this `SetupModel` to `SessionInfo` or `SessionDetails` and keep the properties as id and data 🤔

Reply:
Done!